packages/terra-i18n/tests/nightwatch/i18n-spec.js

```diff
@@ -12,6 +12,8 @@ const defaultTranslations = enTransaltions;
 
 const screenshot = require('terra-toolkit').screenshot;
 
+const pauseInms = 1000;
```

Comment:
The `pauseInms` is pause in millisecond. The `1000ms` value is common used in nightwatch test examples. It works in all 10 tests in local.

There is no API like `waitForElmenentUpdate`. So use `pause()` function in here.